src/environments/environment.prod.ts

```diff
@@ -1,3 +1,8 @@
 export const environment = {
-  production: true
+  production: true,
+  hostname: 'identity.bitclout.com',
```

Comment:
I think this file is just overrides so I don't think you need to define these values twice

Reply:
I believe otherwise, the `environment.prod.ts` file will be used while building with the `--prod` flag and the other file will be used while developing locally.
For example, a developer might have configured the identity service to `identity.bitclout.me` on his local machine, while the production server uses `identity.bitclout.com`; this is a common use-case while developing locally.